deploy/openshift/operator.yaml

```diff
@@ -20,12 +20,31 @@ spec:
         name: deployment-validation-operator
         app: deployment-validation-operator
     spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                - key: app
+                  operator: In
+                  values:
+                  - deployment-validation-operator
+              topologyKey: kubernetes.io/hostname
+            weight: 100
       containers:
-      - image: quay.io/deployment-validation-operator/dv-operator:latest
+      - image: quay.io/deployment-validation-operator/dv-operator:0.1.1
```

Comment:
Not necessarily related to this topic, but we'll need some kind of documented release process/make target that will update this to latest release.  Maybe better to templatize this.